src/Features/Core/Portable/DocumentationComments/CodeFixes/AbstractRemoveDocCommentNodeCodeFixProvider.cs

```diff
@@ -29,31 +31,25 @@ internal abstract class AbstractRemoveDocCommentNodeCodeFixProvider<TXMLElement>
             {
                 context.RegisterCodeFix(
                     new MyCodeAction(
-                        c => RemoveDuplicateParamTag(context.Document, context.Span, c)),
+                        c => RemoveDuplicateParamTagAsync(context.Document, context.Span, c)),
```

Comment:
Whoops, artifact from when I was trying to work out whether to just pass the root and discovered node in and fix it non-async.  :+1: